packages/contracts-bedrock/src/dispute/OutputBisectionGame.sol

```diff
@@ -173,7 +180,7 @@ contract OutputBisectionGame is IOutputBisectionGame, Clone, ISemver {
         // SAFETY:    While the `attack` path does not need an extra check for the post
         //            state's depth in relation to the parent, we don't need another
         //            branch because (n - n) % 2 == 0.
-        bool validStep = VM.step(_stateData, _proof, uuid) == Claim.unwrap(postState.claim);
+        bool validStep = VM.step(_stateData, _proof, Hash.unwrap(uuid)) == Claim.unwrap(postState.claim);
```

Comment:
Will we need to translate the position to account for the split game before passing it to the VM or is that done elsewhere? Also doesn't need to be handled in this PR.

Reply:
We shouldn't - The search can't escape the execution trace bisection sub-tree:
![image](https://github.com/ethereum-optimism/optimism/assets/8406232/1c113dc8-b370-412d-ad20-e05db09b4e97)

Reply:
I was meaning that if the claim is the root of the execution trace that would be at `SPLIT_DEPTH` (maybe +1) but as far as the VM is concerned it's actually the root node of the tree.  So I was expecting that we'd need to translate the position to be relative to the execution trace root before passing it to the VM.

Reply:
I think you can ignore me on this.  We're not passing the claim position to the VM at all on the contract side because it's just given the preimage and we validate that only by checking that it's the pre-image.  We don't need to check that it's at the appropriate trace index.  Hence no need to translate the position like we need to on the challenger side.

Reply:
Ah great callout - yes, this should already be covered. The VM only cares about the UUID passed (which is a commitment to the starting/disputed claim values + positions, or only the disputed claim value + position in the case of the `GENESIS -> GENESIS + 1` trace), and this should be constant for all nodes within an instance of an execution trace subtree. As far as the native VM is concerned, there is only one set of local keys, but in the Solidity version, it's able to accept a `_localIdentifier` argument (the UUID) so that it can read from multiple sets of local keys belonging to the same `OutputBisectionGame`.

See: https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/cannon/MIPS.sol#L203-L208